ftpretty.py

```diff
@@ -223,7 +256,24 @@ def descend(self, remote, force=False):
                     self.conn.mkd(directory)
                     self.conn.cwd(directory)
         return self.conn.pwd()
+    
+    def ascend(self, steps=1):
+        for _ in range(steps):
+            if self.conn.pwd() == "/":
+                break
+            self.conn.cwd("..")
+        return self.conn.pwd()
 
+    def open(self, remote):
```

Comment:
Opening and reading are 2 different things, this actually reads the file.

There are plenty of python options for reading a file.

This seems unneccesary for this API and could easily be done in the client code, suggest removing this.